### PR TITLE
fix: replace hardcoded localhost:9999 with browser origin

### LINKED ISSUE #12336: Update all hardcoded "locahost:9999" in the UI for cloud
There are various locations in the UI where we hardcoded localhost:9999 which we need to update to use the proper url for the running instance.

### PR DESCRIPTION
**Pull request by 121watts**
Closes #12336

### CODE REVIEW

File: ui/src/dataLoaders/actions/dataLoaders.ts [hunk 1]
```diff
@@ -327,7 +327,7 @@ export const createOrUpdateTelegrafConfigAsync = () => async (
     name: TelegrafPluginOutputInfluxDBV2.NameEnum.InfluxdbV2,
     type: TelegrafPluginOutputInfluxDBV2.TypeEnum.Output,
     config: {
-      urls: ['http://127.0.0.1:9999'],
+      urls: [`${window.location.origin}`],
```
Comment (OfTheDelmer) [inline ui/src/dataLoaders/actions/dataLoaders.ts]: nice

Comment (bthesorceror) [reply ui/src/dataLoaders/actions/dataLoaders.ts]: I don't think will work in cloud, we need a specify an exact url. Also will this work in a unit test for the action?

File: ui/src/dataLoaders/components/verifyStep/TelegrafInstructions.tsx [hunk 2]
```diff
@@ -55,6 +56,10 @@ class TelegrafInstructions extends PureComponent<Props> {
       </div>
     )
   }
+
+  private get origin(): string {
+    return window.location.origin
```
Comment (bthesorceror) [inline ui/src/dataLoaders/components/verifyStep/TelegrafInstructions.tsx]: I don't think will work in cloud, we need to specify an exact url.

File: ui/src/dataLoaders/constants/pluginConfigs.ts [hunk 3]
```diff
@@ -29,7 +29,7 @@ import {
   TelegrafPluginInputTail,
 } from '@influxdata/influx'
 
-export const QUICKSTART_SCRAPER_TARGET_URL = 'http://localhost:9999/metrics'
+export const QUICKSTART_SCRAPER_TARGET_URL = `${window.location.origin}/metrics`
```
Comment (bthesorceror) [inline ui/src/dataLoaders/constants/pluginConfigs.ts]: same as above.

File: ui/src/organizations/components/CreateScraperOverlay.tsx [hunk 4]
```diff
@@ -57,7 +57,7 @@ class CreateScraperOverlay extends PureComponent<Props, State> {
       scraper: {
         name: 'My Cool Scraper',
         type: ScraperTargetRequest.TypeEnum.Prometheus,
-        url: 'http://localhost:9999/metrics',
+        url: `${this.origin}/metrics`,
```
Comment (bthesorceror) [inline ui/src/organizations/components/CreateScraperOverlay.tsx]: Same as above.

File: ui/src/organizations/components/CreateScraperOverlay.tsx [hunk 5]
```diff
@@ -166,6 +166,10 @@ class CreateScraperOverlay extends PureComponent<Props, State> {
       notify(scraperCreateFailed())
     }
   }
+
+  private get origin(): string {
+    return window.location.origin
```
Comment (bthesorceror) [inline ui/src/organizations/components/CreateScraperOverlay.tsx]: Same as above.

File: ui/src/dataLoaders/components/verifyStep/TelegrafInstructions.tsx (hunk 2 above)
Comment (121watts) [reply ui/src/dataLoaders/components/verifyStep/TelegrafInstructions.tsx]: @goller says this should work just fine for now in cloud

**Pull request merged**